src/libs/actions/PaymentMethods.js

```diff
@@ -14,7 +14,7 @@ function getPaymentMethods() {
     })
         .then((response) => {
             Onyx.multiSet({
-                [ONYXKEYS.USER_WALLET]: lodashGet(response, 'userWallet', null),
+                [ONYXKEYS.USER_WALLET]: lodashGet(response, 'userWallet', {}),
```

Comment:
Just curious, any reason you decided to default this from `null` to `{}`? AFAIK it doesn't really change anything if we left it as `null`.

Reply:
I'm checking for `_.isEmpty` in `CurrentWalletBalance` so it just made more sense to me /shrug